dll/win32/comctl32/tooltips.c

```diff
@@ -775,7 +817,11 @@ TOOLTIPS_Show (TOOLTIPS_INFO *infoPtr, BOOL track_activate)
         {
           pts[0].x = ptfx;
           pts[0].y = 0;
+          #ifdef __REACTOS__
```

Comment:
```suggestion
#ifdef __REACTOS__
```
Please wait with the merging! These were not here yet when I watched and approved the PR last time!
Please move all of them and the #else and #endif s everywhere to column 0!